assertk/src/commonTest/kotlin/test/assertk/assertions/IterableTest.kt

```diff
@@ -224,8 +223,20 @@ class IterableTest {
         )
     }
 
+    @Test fun partial_matching_content_fails() {
+        val error = assertFails {
+            assertThat(listOf(1, 2, 3) as Iterable<Int>).none { it.isLessThan(2) }
+        }
+        assertEquals(
+            """expected none to pass (2 failures)
+                |${"\t"}${opentestPackageName}AssertionFailedError: expected [[1]] to be less than:<2> but was:<2> ([1, 2, 3])
+                |${"\t"}${opentestPackageName}AssertionFailedError: expected [[2]] to be less than:<2> but was:<3> ([1, 2, 3])
+               """.trimMargin().lines(), error.message!!.lines()
```

Comment:
Hm, error message is confusing here, we'd want the inverse right?

Reply:
I agree on that. I'll have another look tomorrow to see how I can override the message and leave out the details.
I don't think there is a good way how to switch the message to contain the inverse message.

Reply:
Yeah, it'll probably require changes to how errors are collected to get the needed info.

Reply:
Just wanted to shortly apologize, that I did not find time yet to have another look :)

Reply:
No worries! wasn't looking over the holidays anyway 😉